CRM/Contribute/BAO/ContributionRecur.php

```diff
@@ -777,6 +777,12 @@ public static function recurringContribution(&$form) {
     CRM_Core_Form_Date::buildDateRange($form, 'contribution_recur_next_sched_contribution_date', 1, '_low', '_high', ts('From'), FALSE, FALSE, 'birth');
     CRM_Core_Form_Date::buildDateRange($form, 'contribution_recur_failure_retry_date', 1, '_low', '_high', ts('From'), FALSE, FALSE, 'birth');
     CRM_Core_Form_Date::buildDateRange($form, 'contribution_recur_cancel_date', 1, '_low', '_high', ts('From'), FALSE, FALSE, 'birth');
+
+    // Add field for contribution status
+    $form->addSelect('contribution_recur_contribution_status_id',
+      array('entity' => 'contribution', 'multiple' => 'multiple', 'context' => 'search', 'options' => CRM_Contribute_PseudoConstant::contributionStatus() )
```

Comment:
See the related test build failure https://test.civicrm.org/job/CiviCRM-Core-PR/17604/checkstyleResult/new/ 

It says ```There should be no white space before a closing ")"``` so I would say that's a style error. As per coding standard there shouldn't be any trailing space before/after parenthesis.

Reply:
LOL my supervisor had his own coding guidelines which goes against the community. It's fixed now.